### ISSUE TITLE
flow-dev-guide: choose a name for the guide

### ISSUE THREAD

**Issue by knolleary** [node-red/node-red.github.io#93]
I'm not sure `Practice` is quite the right title for this new section. I'd suggest `Developing Flows` would be more consistent with the other sections.

No need to change anything right away - as discussed, once we have more content in the guide I think it will be easier to identify the right way to integrate it with the existing docs. What you have here is a good start begin collaboration on it.

/cc @LEEHYUKJOO

**Comment by LEEHYUKJOO** [comment]
Actually, `Developing Flows` is nice for consistent with other sections. However, I guess that readers can misunderstand that just a way to create flows with "Node-RED" (i.e., how to use Node-RED) is written. The purpose of this guideline is creating flows with readibility, reusability and so on. So, I'd like to consider title more.  How about bellow alternatives. * Developing Effective Flows * Developing Better Flows * Effective Flows

However, above two are too long to be on 1 line (please see an attached caption).
Note that, the height of square is dependent on the number of lines.

@knolleary

**Comment by knolleary** [comment]
I don't think it would be a bad thing to have a single `Developer Flows` guide that covers the basics *and* covers best practices. Otherwise we'd have related content split across different parts of the documentation and a user would need to understand that separation. Not suggestion you need to produce that basic content as well - we can contribute those parts - but from an overall structure, maybe they should live together.

**Comment by LEEHYUKJOO** [comment]
I forgot that the contents of the guideline could be added continuously by other users. I realized that 'Developing Flows' is the best title of this guideline.
